### ISSUE TITLE
Invalid position given by 'W038'

### ISSUE THREAD

**Issue by VinMags16** [AtomLinter/linter-jshint#463]
JSHint returned a point that did not exist in the document being edited.
Rule: `W038`
Requested point: 63:44

Debug information:
```
Atom version: 1.18.0
linter-jshint version: v3.1.5
JSHint version: jshint v2.9.5
Hours since last Atom restart: 0.2
Platform: win32
Current file's scopes: [
  "source.js",
  "punctuation.definition.function.body.begin.bracket.curly.js"
]
linter-jshint configuration: {
  "executablePath": "",
  "lintInlineJavaScript": false,
  "disableWhenNoJshintrcFileInPath": false,
  "scopes": [
    "source.js",
    "source.js-semantic"
  ],
  "jshintFileName": ".jshintrc",
  "jshintignoreFilename": ".jshintignore"
}
```

I'm not sure what this means, I've seen the similar errors for 'W033' but I don't know if it's the same issue.  The error shows at the very top of my JS file only on the first line, regardless of what line of code is there.

**Comment by Arcanemagus** [comment]
There's a 99% chance it's the same root JSHint bug if you are using tabs for indentation. If you _aren't_ then it's an entirely new bug in JSHint that needs to be tracked down and reported 😉.

**Comment by Arcanemagus** [comment]
Assuming that you are indenting your file with tabs this is another case of https://github.com/jshint/jshint/issues/3151. There are reports that if you set your `indent` property of your JSHint configuration to `1` this issue can be worked around. If you don't have a `.jshintrc` file the following should work:
```json
{
  "indent": 1
}
```

If anyone is directed here and _isn't_ using tabs to indent please provide code to reproduce this so an issue can be filed on JSHint to get this fixed there.
